curriculum/challenges/english/20-upcoming-python/learn-tree-traversal-by-building-a-binary-search-tree/655a45c52fa3ea5ece3034c7.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 655a45c52fa3ea5ece3034c7
+title: Step 27
+challengeType: 20
+dashedName: step-27
+---
+
+# --description--
+
+Inside the `delete` method, delete `pass` and call the private helper method `_delete` with the BST root and the key to delete as the arguments.
```

Comment:
```suggestion
Inside the `delete` method, delete `pass` and call the private helper method `_delete` with the root of the binary search tree (BST) and the key to delete as the arguments.
```
I would suggest introducing the acronym gradually by including the full name and BST to help learners remember the meaning of BST.